src/helpers/getResultStringForDictionaryKey.ts

```diff
@@ -0,0 +1,17 @@
+import { ResultStringForDictionaryKey, SwaggerProps } from '../types';
+import { getDescription } from './getDescription';
+import { parseRefType } from './parseRefType';
+
+export const getResultStringForDictionaryKey = ({
```

Comment:
we didn't have an helper before, this code was found here - https://github.com/LandrAudio/openapi-codegen-typescript/pull/53/files#diff-2df477f1aaf80cfbe0f1dfdd0f04f882bc895bd3bcec81e55658851864eaa519L261